crmsh/bootstrap.py

```diff
@@ -1725,11 +1740,17 @@ def join_ssh(seed_host):
             continue
 
         if os.path.exists(os.path.join("/root/.ssh", key)):
-            if not confirm("/root/.ssh/%s already exists - overwrite?" % (key)):
+            # not yes or not True  ==> False, overwrite
+            # not yes or not False ==> True, continue
+            # not no or not True ==> True, continue
+            # not no or not False  ==> True, continue
+            if not confirm("/root/.ssh/%s already exists - overwrite?" % (key)) or not need_pw:
```

Comment:
I don't understand this `confirm` question. overwrite what? The authorized_keys content?

Reply:
Overwrite the ssh key if confirmed `yes`, so we need add `need_pw` flag here, to avoid overwrite if already passwordless